application/src/main/java/de/wuespace/telestion/project/daedalus2/RawFileLogger.java

```diff
@@ -0,0 +1,40 @@
+package de.wuespace.telestion.project.daedalus2;
+
+import de.wuespace.telestion.api.verticle.GenericConfiguration;
+import de.wuespace.telestion.api.verticle.TelestionVerticle;
+import de.wuespace.telestion.api.verticle.trait.WithEventBus;
+import io.vertx.core.Promise;
+import io.vertx.core.buffer.Buffer;
+import io.vertx.core.file.AsyncFile;
+import io.vertx.core.file.FileSystem;
+import io.vertx.core.file.OpenOptions;
+
+/**
+ * It writes the raw mavlink messages to a file
+ */
+public class RawFileLogger extends TelestionVerticle<GenericConfiguration> implements WithEventBus {
+	AsyncFile file;
+
+	@Override
+	public void onStart() throws Exception {
+		// It creates a file and opens it in blocking mode.
+		FileSystem fs = getVertx().fileSystem();
+		file = fs.openBlocking("data/out.bin", new OpenOptions()
+				.setAppend(true)
+				.setCreate(true)
+				.setSync(true)
+		);
```

Comment:
Why using the blocking version instead of the `vertx.filesystem().open(filepath, options)` version?

Reply:
Because I didn't want to deal with asynchronous code (for readability purposes) if not necessary. And opening / creating if it doesn't exist a file shouldn't take so long that this creates any bigger issues.